packages/aws-cdk-lib/pipelines/lib/codepipeline/private/codebuild-factory.ts

```diff
@@ -286,7 +286,7 @@ export class CodeBuildFactory implements ICodePipelineActionFactory {
       buildSpecString: actualBuildSpec.toBuildSpec(),
     }));
 
-    const actionName = options.actionName ?? this.stepId;
```

Comment:
This never did anything. The type of `actionName` is `string` (not `string | undefined`).